addon_imps/storage/owncloud.py

```diff
@@ -145,12 +145,15 @@ async def build_wb_config(self) -> dict:
         root_host = f"{parsed_url.scheme}://{parsed_url.netloc}"
         folder_path = ""
 
-        wb_config = {
+        if self.config.connected_root_id:
+            _, subpath = _parse_item_id(self.config.connected_root_id)
+            folder_path = subpath.strip("/") or ""
```

Comment:
nit: Worst case scenarion of `"".strip("/")` or  `"/".strip("/")` is `""`
```suggestion
            folder_path = subpath.strip("/")
```